### DISCUSSION TITLE
How do you make etching acid?

### DISCUSSION THREAD

**Discussion by IgnisAstra** [TeamPneumatic/pnc-repressurized#1135] [Q&A] [answered]
Can't find the proper recipe anywhere, or maybe im blind and don't see it.

**Comment by desht** [comment, marked as answer]
Check the recipe for "Etching Acid Bucket".  The default recipe makes a bucket of the acid.  If it's been changed in your modpack, ask the modpack maker.

![image](https://user-images.githubusercontent.com/736831/215704560-202c3df3-3050-4e94-8c46-37b9f6ab2727.png)

**Reply by IgnisAstra** [reply]
Thank you, That legit never showed up on the modpack. I guess it was updated recently or something got turned on.
